New entries in UI and UX -- fixing PR (#4)
js6450/docs: content/uiux/concepts/divergent-thinking/divergent-thinking.md

---
Title: 'Divergent Thinking'

> Description: 'Divergent thinking is a creative mode of thinking that explores many possible solutions and generates novel ideas.'

Reviewer: ```suggestion
Description: 'Divergent thinking is a creative mode of thinking that explores many possible solutions and seeks to generate novel ideas.'
```

Reviewer: Perhaps this is a bit of a minor quibble, but I think we should avoid the implicit suggestion that engaging in the process will inevitably lead to novel ideas. That is the aim of the process not a consequence of simply using this framework

Reviewer 2: cc: @lucwhite

Reviewer 3: Yeah, I suppose this would be more clear. We can make this change.

Subjects:
  - 'Web Design'
Tags:
  - 'UI'
  - 'UX'
CatalogContent:
  - 'intro-to-ui-ux'
  - 'paths/front-end-engineer-career-path'
---

> **Divergent thinking** is a creative mode of thinking that explores many possible solutions and generates novel ideas.

Reviewer: same as above

Reviewer 3: ```suggestion
**Divergent thinking** is a creative mode of thinking that explores many possible solutions and seeks to generate novel ideas.
```

> Design activities that encourage divergent thinking often emphasize spontaneity and playfulness, and aim to create an open atmosphere for sharing good and bad ideas. Activities could include brainstorming, free writing, and bubble mapping. Once divergent thinking methods are used to generate a wide range of ideas, [convergent thinking](https://www.codecademy.com/resources/docs/uiux/convergent-thinking) can be used to hone in on ideas and make decisions.

Reviewer: ```suggestion
Design activities that encourage divergent thinking often emphasize spontaneity and playfulness, and aim to create an open atmosphere for sharing good and bad ideas. Activities could include brainstorming, free writing, and bubble mapping. Once divergent thinking methods are used to generate a wide range of ideas, [convergent thinking](https://www.codecademy.com/resources/docs/uiux/convergent-thinking) can be used to home in on ideas and make decisions.
```

Reviewer: I think as structured the correct word would be 'home'. But I imagine it could also be written as 'used to hone ideas and make decisions'

Reviewer 3: I think "[hone in on](https://www.merriam-webster.com/dictionary/hone%20in%20on#:~:text=Definition%20of%20hone%20in%20on&text=%3A%20to%20find%20and%20go%20directly,error%20for%20home%20in%20on.)" is actually the correct phrase

Reviewer 3: [Weird](https://www.merriam-webster.com/dictionary/home%20in%20on#:~:text=Definition%20of%20home%20in%20on,the%20cause%20of%20the%20disease.). Both right, I guess?

> Design activities that encourage divergent thinking often emphasize spontaneity and playfulness, and aim to create an open atmosphere for sharing good and bad ideas. Activities could include brainstorming, free writing, and bubble mapping. Once divergent thinking methods are used to generate a wide range of ideas, [convergent thinking](https://www.codecademy.com/resources/docs/uiux/convergent-thinking) can be used to home in on ideas and make decisions.

Reviewer 4: ```suggestion
Design activities that encourage divergent thinking often emphasize spontaneity and playfulness, and aim to create an open atmosphere for sharing good and bad ideas. Activities could include brainstorming, free writing, and bubble mapping. Once divergent thinking methods are used to generate a wide range of ideas, [convergent thinking](https://www.codecademy.com/resources/docs/uiux/convergent-thinking) can be used to hone in on ideas and make decisions.
```

> Many design methodologies explicitly define stages where divergent thinking is the focus. For example, in the [double diamond diagram](https://www.codecademy.com/resources/docs/uiux/double-diamond-diagram), the "Discover" stage focuses on divergent strategy, or exploring the problem and landscape, and the "Develop" stage focuses on divergent execution, or generating a range of ideas and solutions.

Reviewer: @Dusch4593 I don't see quotes addressed in the style guide. My interpretation would suggest that "Discover" and "Develop" should be in bold in lieu of quotes

Reviewer 2: [**Bolding**](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/#bold) is more for new terms, keywords, or concepts (such as **Divergent thinking** for this entry). I suggest either using _italics_ or no emphasis or quotes at all.

Reviewer 3: ```suggestion
Many design methodologies explicitly define stages where divergent thinking is the focus. For example, in the [double diamond diagram](https://www.codecademy.com/resources/docs/uiux/double-diamond-diagram), the Discover stage focuses on divergent strategy, or exploring the problem and landscape, and the Develop stage focuses on divergent execution, or generating a range of ideas and solutions.
```

> Many design methodologies explicitly define stages where divergent thinking is the focus. For example, in the [double diamond diagram](https://www.codecademy.com/resources/docs/uiux/double-diamond-diagram), the Discover stage focuses on divergent strategy, or exploring the problem and landscape, and the Develop stage focuses on divergent execution, or generating a range of ideas and solutions.

Reviewer 4: ```suggestion
Many design methodologies explicitly define stages where divergent thinking is the focus. For example, in the [double diamond diagram](https://www.codecademy.com/resources/docs/uiux/double-diamond-diagram), the "Discover" stage focuses on divergent strategy or exploring the problem and landscape, and the "Develop" stage focuses on divergent execution or generating a range of ideas and solutions.
```
